pkg/core/transaction/transaction_test.go

```diff
@@ -321,3 +321,28 @@ func BenchmarkTxHash(b *testing.B) {
 		_ = tx.Hash()
 	}
 }
+
+func TestTransaction_DeepCopy(t *testing.T) {
+	origTx := New([]byte{0x01, 0x02, 0x03}, 1000)
+	origTx.NetworkFee = 2000
+	origTx.Signers = []Signer{{Account: random.Uint160()}}
+	origTx.Scripts = []Witness{
+		{
+			InvocationScript:   []byte{0x04, 0x05},
+			VerificationScript: []byte{0x06, 0x07},
+		},
+	}
+
+	copyTx := origTx.Copy()
```

Comment:
Not all branches are covered here.

Reply:
Still not resolved. We need transaction with all fields filled, including attributes, signers with scopes, rules, etc.

Reply:
is it resolved?